apps/prairielearn/src/ee/pages/instructorInstanceAdminLti13/instructorInstanceAdminLti13.ts

```diff
@@ -253,6 +263,29 @@ router.post(
         }
       });
       return res.redirect(`/pl/jobSequence/${serverJob.jobSequenceId}`);
+    } else if (req.body.__action === 'send_grades') {
+      // Should something like this be in a model?
+      const assessment = await queryRow(
+        sql.select_assessment_with_course_instance,
+        {
+          assessment_id: req.body.unsafe_assessment_id,
```

Comment:
In PT we've established the convention that we name the SQL parameter `unsafe_*` in this case; this makes it easy to tell during review that a query is using an unsafe value so one can check that membership checks/etc. are being enforced.